xml/update-alternatives.xml

```diff
@@ -428,38 +437,39 @@ foo - auto mode
     <command>/usr/local/bin/foo</command>.
    </para>
    <para>
-    If needed, you can install additional alternatives. 
+    If needed, you can install additional alternatives.
     To remove an alternative, use the following command:
    </para>
-   <screen>&prompt.root;<command>update-alternative</command> --remove foo /usr/local/bin/foo-2</screen>
+   <screen>&prompt.root;<command>update-alternatives</command> --remove foo /usr/local/bin/foo-2</screen>
    <para>
     After this script has been removed, the &as; for the foo group looks
     like this:
    </para>
-   <screen>&prompt.root;<command>update-alternative</command> --display foo
+   <screen>&prompt.root;<command>update-alternatives</command> --display foo
 foo - auto mode
   link best version is /usr/local/bin/foo-3
   link currently points to /usr/local/bin/foo-3
   link foo is /usr/local/bin/foo
 /usr/local/bin/foo-3 - priority 300</screen>
   </sect1>
 
-  <sect1 xml:id="sec.ua.slaves">
+  <sect1 xml:id="sec.ua.slave">
    <title>Defining Dependent Alternatives</title>
    <para>
-    If you have alternatives, the script itself is not enough. Usually
-    scripts comes with a manpage or some other configuration options.
-    To create alternatives which are dependent from a master
-    link you need to create <emphasis>slave alternatives</emphasis>.
+    If you have alternatives, the script itself is not enough. Most commands
```

Comment:
"the script itself is not enough" sounds weird, as if this was an inevitable thing though it isn't. maybe: "Setting up alternatives for a script is often not enough."